apps/store/extensions/assets/default/pages/list.jag

```diff
@@ -62,9 +62,29 @@ require('/modules/store.js').exec(function(ctx) {
     PAGING.count = (count || DEFAULT_LIST_PAGIN_COUNT);
     PAGING.paginationLimit = (paginationLimit || constants.DEFAULT_ASSET_PAGIN.paginationLimit);
 
+    var removeCategorizationFromHistory = function(ctx, options) {
+        var tenantContext = tenantApi.createTenantAwareAssetResources(ctx.session, options).context;
+        var categorizationFields = tenantContext.rxtManager.listRxtCategoryFields(tenantContext.assetType, "options");
+        categorizationFields.push({"name" : {"fullName" : "taxonomy"}});
+        categorizationFields.push({"name" : {"fullName" : "mediaType"}});
+        var queryParams = q.split(",");
+        for(var i=0; i < queryParams.length; ++i){
+            var queryParam = queryParams[i];
+            for (var index=0; index < categorizationFields.length; index++) {
```

Comment:
This method removes the categorization and taxonomy fields from the search query since it needs to be removed from saving in the search history